db/migrate/20161220020802_add_column_to_site.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToSite < ActiveRecord::Migration
+  def change
+    add_column :sites, :resize_thumb, :boolean, default: false, null: false
```

Comment:
this column was for some kind like on/off settings by site, 
though i changed my mind to remove this and created new PR at #693